City-of-Helsinki/helsinki-paatokset
# PP-188: Helsinki Kanava -integration

- Added new migration paatokset_meeting_videos
- Added latest recording section to policymaker template
- Added new setting to identify which policymaker node is city council

To test: 

- Add HELSINKI_KANAVA_ID and HELSINKI_KANAVA_SECRET variables to your.env. Values can be found here: https://helsinkisolutionoffice.atlassian.net/wiki/spaces/PP/pages/2689106104/Helsinki-kanavan+videot (public repo so I won't post them here)
- Run `docker-composer up --build -d app` to rebuild container
- Run `printenv` inside container to verify your newly added variables are loaded
- Run `make drush-cim && make drush-cr`. Check that `paatokset_helsinki_kanava` -module is active. 
- Go to https://helsinki-paatokset.docker.so/fi/admin/config/helsinki-kanava/settings. Set the **City council** setting to point to your local node of City Council (Kaupunginvaltuusto).  All mettings link should be correct from cim.
- Run `drush mim paatokset_meeting_videos`
- View your City Council node. Video embed should be printed to the bottom of the page.
